googletest/src/gtest.cc

```diff
@@ -3217,8 +3217,7 @@ static WORD GetNewColor(GTestColor color, WORD old_color_attrs) {
 
 #else
 
-// Returns the ANSI color code for the given color. GTestColor::kDefault is
-// an invalid input.
+// Returns the ANSI color code for the given color.
```

Comment:
I think the code fix for the warning is fine, but I don't want to expand the support for kDefault. That is to say, we should keep the current contract for this function. Can we add an `assert(false)` on line 3231 before returning "9"?

Reply:
ping @pgawro

Reply:
@okias Right I made code changes but forget about comment. Sorry for that. Now it's up to @derekmauro since requested changes are made